performance_manager/load_vp_events.py

```diff
@@ -0,0 +1,289 @@
+    },
+    {
+        "path": "mbta-ctd-dataplatform-dev-springboard/lamp/RT_VEHICLE_POSITIONS/year=2022/month=7/day=20/hour=9/647c1c01e703466fb9679a380ad89ac4-0.parquet"
+    },
+    {
+        "path": "mbta-ctd-dataplatform-dev-springboard/lamp/RT_VEHICLE_POSITIONS/year=2022/month=7/day=20/hour=9/6635c9551e4b4c2ca64152bd167f751a-0.parquet"
+    },
+]
+
+
+def load_environment() -> None:
+    """
+    boostrap .env file for local development
+    """
+    try:
+        if int(os.environ.get("BOOTSTRAPPED", 0)) == 1:
+            return
+
+        here = os.path.dirname(os.path.abspath(__file__))
+        env_file = os.path.join(here, "..", ".env")
+        logging.info("bootstrapping with env file %s", env_file)
+
+        with open(env_file, "r", encoding="utf8") as reader:
+            for line in reader.readlines():
+                line = line.rstrip("\n")
+                line.replace('"', "")
+                if line.startswith("#") or line == "":
+                    continue
+                key, value = line.split("=")
+                logging.info("setting %s to %s", key, value)
+                os.environ[key] = value
+
+    except Exception as exception:
+        logging.error("error while trying to bootstrap")
+        logging.exception(exception)
+        raise exception
+
+
+def parse_args(args: List[str]) -> argparse.Namespace:
+    """parse args for running this entrypoint script"""
+    parser = argparse.ArgumentParser(description=DESCRIPTION)
+    parser.add_argument(
+        "--experimental",
+        action="store_true",
+        dest="experimental",
+        help="if set, use a sqllite engine for quicker development",
+    )
+    parser.add_argument(
+        "--seed",
+        action="store_true",
+        dest="seed",
+        help="if set, seed metadataLog table with paths",
+    )
+
+    return parser.parse_args(args)
+
+
+def get_db_engine(
+    experimental: bool, seed: bool
+) -> sqlalchemy.orm.sessionmaker:
+    """
+    create database orm objects, return engine for db operations
+    """
+    if experimental:
+        sql_engine = get_experimental_engine(echo=True)
+    else:
+        sql_engine = get_local_engine(echo=True)
+
+    SqlBase.metadata.drop_all(sql_engine)
+    SqlBase.metadata.create_all(sql_engine)
+
+    db_session = sessionmaker(bind=sql_engine)
+
+    if seed:
+        with db_session.begin() as cur:  # type: ignore
+            cur.execute(insert(MetadataLog.__table__), LOAD_PATHS)
+            cur.commit()
+
+    return db_session
+
+
+if __name__ == "__main__":
+    load_environment()
+
+    parsed_args = parse_args(sys.argv[1:])
+
+    session = get_db_engine(parsed_args.experimental, parsed_args.seed)
+
+    """
+    pull list of objects that need processing from metadata table
+    group objects by similar hourly folders
+    """
+    read_md_log = select((MetadataLog.id, MetadataLog.path)).where(
+        MetadataLog.processed == 0
```

Comment:
i had to change this to `== False` on postgres.

Reply:
Incorporated